new/t1.rb

```diff
@@ -0,0 +1,11 @@
+class Plumbus
+private
+  def smooth; end
+end
+
+
+a = [1, 2, 3,
+  4, 5, 6]
```

Comment:
Align the elements of an array literal if they span more than one line.

```suggestion
     4, 5, 6]
```